### DISCUSSION TITLE
company logo graphic

### DISCUSSION THREAD

**Discussion by Tryglav** [neos-sdi/adfsmfa#280] [Q&A] [answered]
Hi
I have a question. Can we disable or change company logo graphic on default email lcid templates, without importing changed templates? Previous versions hasn't company logo graphic by default.

![Capture](https://user-images.githubusercontent.com/63842532/215765662-acacb78f-15ab-42d2-ad47-c978cf5c3770.PNG)

**Comment by redhook62** [comment, marked as answer]
Yes, of course.
This logo is a default.

You can put your own logo by copying it to \ProgramFiles\MFA\Config\logo.png

This will not be replaced by a new update.

If this image file is not present, there will be no logos in the emails, however during a future update, the Neos one will be in place.
We will update the documentation.

regards

**Reply by Tryglav** [reply]
Hi @redhook62 
Thank you for explanation.
regards
